aws-logs-logstream/aws-logs-logstream.json

```diff
@@ -0,0 +1,43 @@
+{
+    "typeName" : "AWS::Logs::LogStream",
+    "description" : "Resource Type definition for AWS::Logs::LogStream",
+    "additionalProperties" : false,
+    "properties" : {
+        "LogGroupName" : {
+            "type" : "string",
+            "description" : "The name of the log group where the log stream is created."
+        },
+        "LogStreamName" : {
+            "type" : "string",
+            "description" : "The name of the log stream. The name must be unique wihtin the log group."
+        }
+    },
+    "required" : [ "LogGroupName", "LogStreamName" ],
```

Comment:
LogStreamName is not a required property. I think the handler has to generate the name if not provided by the user. 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-logstream.html#aws-resource-logs-logstream-properties